bundled/kokkos-4.5.01/tpls/mdspan/include/experimental/__p2630_bits/submdspan_mapping.hpp

```diff
@@ -168,7 +168,7 @@ struct deduce_layout_left_submapping<
   }
 
   MDSPAN_INLINE_FUNCTION
-  constexpr static bool layout_left_padded_value() {
+  constexpr bool layout_left_padded_value() {
```

Comment:
This change was not done upstream. We should probably do it there too.

Reply:
Oh, that must have slipped through the cracks of the split of https://github.com/kokkos/kokkos/pull/8071 into the Kokkos and mdspan parts. Would you like me to write a patch for mdspan?

In any case, I think it is the right thing to do and I would like to leave it in here if you wouldn't mind.

Reply:
No, scratch that. This is a member function. I made that change by mistake. This is what @masterleinad pointed out in https://github.com/kokkos/kokkos/pull/8071#pullrequestreview-2833203465. I'm going to undo this part of the patch of course!